flexmeasures/ui/templates/base.html

```diff
@@ -11,6 +11,9 @@
     </title>
     {% endblock head %}
     <meta charset="windows-1252">
+    {% if flask_env != 'development' and flask_env != 'testing' %}
```

Comment:
Why not in development, and why not in testing (separate questions)?

Reply:
Because we're developing on http, so requesting https (in the usual case, not your corner case) is likely to fail

Reply:
In development, we have hosted everything locally, so that's why https requests would fail to show any content.

Reply:
Tests also run http, so I'll go with the same answer

Reply:
Got it, thanks!